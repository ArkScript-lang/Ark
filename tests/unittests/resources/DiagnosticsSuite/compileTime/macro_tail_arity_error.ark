($ a (tail 1 2))
(print a)
