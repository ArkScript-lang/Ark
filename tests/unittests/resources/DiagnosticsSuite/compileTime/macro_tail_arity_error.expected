At (tail 1 2) @ 2:10
    1 | ($ a (tail 1 2))
    2 | (print a)
      | ^~~~~~~~~
    3 |
      | ^
        When expanding `tail' inside a macro, got 2 arguments, expected 1
